ledger-api/grpc-definitions/com/daml/ledger/api/v1/events_service.proto

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+/*
+Satisfied Design goals:
+- can co-exist with existing API services for single-domain participants
+- can be served with the 1.17 IndexDB
+- supports serving a snapshot and delta on the same stream
+- serves transaction nodes in individual messages
+- supports retrieving the transaction of a transaction node
+- supports consumers in building the transaction tree
+
+WIP:
+- supports TransactionFilter
+- provides clear docs for how the events are filtered
+- minimize differences to existing transaction_service and active_contracts_service
+- ensure that there is an efficient strategy for serving the stream
+
+Next design goals:
+- support evolving the interface to v2 multi-domain transactions
+- include completions
+- communicate pruning events
+- include all data required for Ledger API completeness
+
+
+*/
+
+
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "google/protobuf/wrappers.proto";
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+
+
+option java_outer_classname = "EventsServiceServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+
+service EventsService {
+  // Get the address of a recent transaction as of which an event source can be provided.
```

Comment:
Does it have to be a transaction? Is "event" not good enough? I imagine the case where there are a lot of non-transaction events and we may want to resubscribe from in between.

Reply:
Discussion outcome: cleanup language around transaction, action, node, event.